src/main/java/backend/model/sponsoring/Sponsoring.kt

```diff
@@ -37,15 +39,18 @@ class Sponsoring : BasicEntity, Billable {
     lateinit var limit: Money
         private set
 
-    @ManyToOne
-    var team: Team? = null
+    @ManyToMany
+    var teams: MutableSet<Team> = HashSet()
```

Comment:
I chose a set for now. We might also want to use a list instead. I want to make sure that each team can only be once in there, so a Set felt better as a starting point.